### ISSUE TITLE
minetestserver restart with builtin command

### ISSUE THREAD

**Issue by siliconsniffer** [minetest/minetest#14759]
### Problem

When changing the config or adding mods to the world, one currently has to shutdown the server and then restart it which takes a bit.

(Though for the config there seems to be [a mod](https://content.minetest.net/packages/Emojiminetest/settings_loader/) and @jordan4ibanez is working on a GUI for minetestsever)

### Solutions

A builtin `restart` command similar to `shutdown`would be nice.

### Alternatives

It could also become a flag of `shutdown`.

**Comment by nininik0** [comment]
great idea! i always wondered why that command wasnt there.........................

**Comment by SwissalpS** [comment]
It would be even nicer if the ``/restart`` command also had an option to only restart once the server is empty. [Similar to this.](https://github.com/pandorabox-io/pandorabox_custom/blob/8cb9874508b6b8e331dace39e57f81e99a53b571/restart_if_empty.lua) (On that server docker currently handles restarting / keeping alive)

**Comment by siliconsniffer** [comment]
Yeah, that's a great idea! One flag to force a restart and one to wait until everyone is disconnected. I think it would make sense for players to always have the option to reconnect - not like the shutdown command where you would have to specify this.

**Comment by jordan4ibanez** [comment]
I wish there was a restart command

**Comment by siliconsniffer** [comment]
I am not sure whether this is a duplicate of #11324 or not.

**Comment by sfan5** [comment]
It is.
